index.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module "pg-range" {
+  import { default as StRange } from "strange";
+
+  // Must use any here as we cannot use a namespace as a type
+  export function install(pg: any): void;
```

Comment:
This could be avoided by requires pg.types be passed instead to work around the issue but would require a breaking change